src/async-wrap.cc

```diff
@@ -150,38 +150,39 @@ static void DestroyIdsCb(uv_timer_t* handle) {
   TryCatch try_catch(env->isolate());
 
   do {
-    std::vector<double> destroy_ids_list;
-    destroy_ids_list.swap(*env->destroy_ids_list());
+    std::vector<double> destroy_async_id_list;
     if (!env->can_call_into_js()) return;
```

Comment:
(conflict no 1)

Reply:
Is the swapping of `can_call_into_js` check and the list swap intentional?

Reply:
@TimothyGu Not really intentional, but I don’t think it matters – that there is a similar check in `PushBackDestroyAsyncId()` means we’re okay with dropping `destroy` callbacks if `can_call_into_js()` is false.

If we were not okay with that, I think this change would make it More Correct™, because right now we *would* silently drop the destroy id queue?

Reply:
Okay, just making sure this wasn't overlooked.

Reply:
Yeah, thanks for pointing to it :)